openml-lightgbm-provider/lightgbmlib_build/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.feedzai.openml.lightgbmlib</groupId>
+  <artifactId>lightgbmlib</artifactId>
+  <version>2.3.1</version>
+  <packaging>jar</packaging>
+  <url>git@gitlab.feedzai.com:data-science/make-lightgbm.git</url>
```

Comment:
The URL isn't correct, this is now on github

Reply:
This is another thing: this repository is the one used to generate a our custom lightgbm jar. It's not a reference to this repository.

Reply:
Should we also place that one on github?

Reply:
I will leave this as it is